source/rust_verify/src/rust_to_vir_expr.rs

```diff
@@ -294,6 +297,10 @@ fn fn_call_to_vir<'tcx>(
     let is_ensures = f_name == "builtin::ensures";
     let is_invariant = f_name == "builtin::invariant";
     let is_decreases = f_name == "builtin::decreases";
+    let is_opens_invariants_none = f_name == "builtin::opens_invariants_none";
+    let is_opens_invariants_any = f_name == "builtin::opens_invariants_any";
+    let is_opens_invariants = f_name == "builtin::opens_invariants";
+    let is_opens_invariants_except = f_name == "builtin::opens_invariants_except";
```

Comment:
Fwiw, there is a more robust way of doing this that doesn't rely on path strings (https://doc.rust-lang.org/nightly/nightly-rustc/rustc_hir/lang_items/index.html), I've just never gotten around to it yet.